server/base/src/main/java/org/apache/accumulo/server/metadata/ConditionalTabletMutatorImpl.java

```diff
@@ -164,6 +175,16 @@ public Ample.ConditionalTabletMutator requireOperation(TabletOperationId opid) {
     return this;
   }
 
+  @Override
+  public Ample.ConditionalTabletMutator requireNextOperation() {
```

Comment:
`requireNextOperation` and `requireAbsentNextOperation` are exactly the same. Is there a way to have a Condition that requires a Value (any Value)?

Reply:
We should probably be passing in an operation id to this method, want to require a specific operation id not any operation id.

Reply:
The following code is in `Manager.requestTabletUnload`:
```
          // If the following conditional mutation fails, but the nextOperationId is set, then
          // this is not a failure condition. The tablet should still be unloaded using the
          // existing nextOperationId
          mutator.mutateTablet(ke).requireAbsentNextOperation().prepareForOperation(tabletOpId)
              .submit(tm -> tm.getNextOperationId() != null);
```

In the case that the conditional mutation fails, it is still "successful" if the nextOperationId is not null. Meaning, some other process put a value in that column first. I made the assumption that this is ok, because we don't necessarily care that this tabletOpId is in the column, we just care that the tablet gets unloaded. Because of this, requireNextOperation does not have a parameter.

Reply:
The concept of a nextOperation has been reverted.